components/browser/session/src/main/java/mozilla/components/browser/session/Download.kt

```diff
@@ -24,5 +27,17 @@ data class Download(
     val contentLength: Long? = null,
     val userAgent: String? = null,
     val destinationDirectory: String = Environment.DIRECTORY_DOWNLOADS,
-    val referrerUrl: String? = null
+    val referrerUrl: String? = null,
+    val id: String = UUID.randomUUID().toString()
```

Comment:
Now that we add an ID here, we should refactor feature-downloads to use this ID (it generates a unique long right now). I'll take another look tomorrow, but we could also do that as a follow-up.

Reply:
Yeah, let's do it in a follow-up when migrating the feature (#4282).